task5/src/main/java/com/example/demo1/GetNameServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.example.demo1;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet(name = "nameServlet", value = "/get")
+public class GetNameServlet extends HttpServlet {
+    private static final String NAME_PARAMETER = "name";
+    private String name;
+
+    public void init() {
+    }
+
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+
+        name = request.getParameter(NAME_PARAMETER);
+        PrintWriter out = response.getWriter();
+        out.println("<html><body>");
+        out.println("<h1>" + name + "</h1>");
```

Comment:
Check for the length of the name missing

Reply:
Okay, thanks. And in the end, what grade do I get for the 5th task?